ludwig/api.py

```diff
@@ -322,6 +323,12 @@ def train(
             dataset is provided it is preprocessed and cached by saving an HDF5
             and JSON files to avoid running the preprocessing again. If this
             parameter is `False`, the HDF5 and JSON file are not saved.
+        :param skip_save_inference_module: (bool, default: `False`) disables
```

Comment:
Comment seems to be outdated, as it looks like the TS model is only saved once after training.

Reply:
Updated the docstring to reflect behavior described in other PR comment.